js/src/api/subscriptions/personal.js

```diff
@@ -37,12 +39,33 @@ export default class Personal {
     ]);
   }
 
+  // FIXME: Because of the different API instances, the "wait for valid changes" approach
+  // doesn't work. Since the defaultAccount is critical to operation, we poll in exactly
+  // same way we do in ../eth (ala same as eth_blockNumber) and update. This should be moved
+  // to pub-sub as it becomes available
   _defaultAccount = () => {
+    const nextTimeout = (timeout = 1000) => {
```

Comment:
It doesn't feel right to make a call every second whereas this shouldn't really change... It's understandable for `eth_blockNumber` because it's more or less 14 useless calls for 1 useful one. But here it's thousands I'd guess.
Not sure what better approach we could have though (except having a proper WS push)...